_includes/tutorials/kafka-connect-datagen-ccloud/kafka/markup/dev/check-connector.adoc

```diff
@@ -0,0 +1,11 @@
+Check the status of the connector:
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-connect-datagen-ccloud/kafka/code/tutorial-steps/dev/check-connector.sh %}</code></pre>
++++++
+
+Wait a minute for the connector to start. It will change state from `PROVISIONING` to `RUNNING` when it is ready.
```

Comment:
FYC
```suggestion
After a minute or two, rerun this command to get the updated Status, it will change from `PROVISIONING` to `RUNNING` when the Connector is ready.
```